server/cmd/code-annotation/main.go

```diff
@@ -68,11 +68,21 @@ func main() {
 	// frontend
 	r.Get("/*", func(w http.ResponseWriter, r *http.Request) {
 		filepath := "build" + r.URL.Path
-		if _, err := os.Stat(filepath); err == nil {
-			http.ServeFile(w, r, filepath)
-			return
+		b, err := Asset(filepath)
+		// fallback on index, will be handled by FE router
```

Comment:
this is a good idea? maybe is better return a 404 if is file not found and a 500 if not

Reply:
it is. Frontend has urls like `/experiment/1/2`, backend knows nothing about them. It's handled correctly if you click on a link (js will catch event), but if you open this url the first time response will be served by the server. That's why we have to return `index.html`. JS there will handle routing correct routing (and show 404 if path doesn't exist). It's a limitation for single page application if you want to have nice urls without `#`.

Reply:
meeeh

Reply:
frontend 😂